pkgs/development/python-modules/mpv-jsonipc/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests
+, tqdm, websocket_client, pytest, pillow, mock, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "mpv-jsonipc";
+  version = "unstable-2020-02-19";
```

Comment:
```suggestion
  version = "unstable-2020-02-19";
  disabled = pythonOlder "3.6";
```
to avoid:
```
  Processing ./python_mpv_jsonipc-1.1.5-py2-none-any.whl
  ERROR: Package 'python-mpv-jsonipc' requires a different Python: 2.7.17 not in '>=3.6'
cannot build derivation '/nix/store/y6499z8wxmlqjxpf8dgwim4pylqa9l7w-plex-mpv-shim-1.7.12.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/akz7a2cxndlbmq4k3hkx6y7lj7318dwl-env.drv': 2 dependencies couldn't be built
[3 built (2 failed), 1 copied (0.2 MiB), 0.0 MiB DL]
error: build of '/nix/store/akz7a2cxndlbmq4k3hkx6y7lj7318dwl-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/77884
2 package failed to build:
plex-mpv-shim python27Packages.mpv-jsonipc
```

Reply:
Updated per your suggestion. Pushed. `nix-review pr 77884` passes now.

Question -- I understand why nix review tested the python36 review, but why does it look like yours try to build plex-mpv-shim with python36 whereas I was using plex-mpv-shim happily (aka, it wasn't building with python36 on my machine).

Reply:
Okay, thanks. `nix-review pr 77884` passes now.

Reply:
I pulled it from their assertion: `Python: 2.7.17 not in '>=3.6'`

Reply:
@jonringer: I've got the pname change staged here locally, do I need to also rename the file?